tools/jenkins/run_full_performance.sh

```diff
@@ -41,23 +41,32 @@ tools/run_tests/run_performance_tests.py \
     --category scalable \
     --bq_result_table performance_test.performance_experiment \
     --remote_worker_host grpc-performance-server-8core grpc-performance-client-8core grpc-performance-client2-8core \
+    --xml_report report_8core.xml \
     || EXIT_CODE=1
 
+# prevent pushing leftover build files to remote hosts in the next step.
+git clean -fdxq --exclude='report*.xml'
```

Comment:
I could maybe see this turning up in an odd way if another reports directory is added, e.g. for the perf reports. Saw something about git-archive, probably outside of this change but maybe it would work to use that to make the .tar?